images/macos/provision/configuration/preimagedata.sh

```diff
@@ -25,8 +25,8 @@ cat <<EOF > $imagedata_file
         "detail": "${os_name}\n${os_version}\n${os_build}"
       },
       {
-        "group": "Runner Image",
```

Comment:
Sorry I still want the group to to say "Runner Image"  - I want line 29 to be called just "Image"

Reply:
Updated. Please review.